presto-main/src/test/java/io/prestosql/sql/planner/TestDynamicFilter.java

```diff
@@ -94,6 +94,37 @@ public void testEmptyJoinCriteria()
                                         tableScan("lineitem")))));
     }
 
+    @Test
+    public void testCrossJoinInequalityDF()
+    {
+        assertPlan("SELECT o.orderkey FROM orders o, lineitem l WHERE o.orderkey < l.orderkey",
+                anyTree(filter("O_ORDERKEY < L_ORDERKEY",
+                        join(
+                                INNER,
+                                ImmutableList.of(),
+                                node(
```

Comment:
You should be able to match for dynamic filter as well using something like ImmutableMap.of("O_COMMENT", "L_COMMENT")